common/web/gesture-recognizer/src/engine/eventSequentializationQueue.ts

```diff
@@ -0,0 +1,51 @@
+import { timedPromise } from "@keymanapp/web-utils";
+
+export class EventSequentializationQueue {
+  private queue: (() => Promise<void> | void)[];
+  private defermentPromise: Promise<void>;
```

Comment:
A promise _is_ a deferment. What is this for? It should be `processNextEventPromise` or something like that.